cli/cmd/lib_cli_config.go

```diff
@@ -144,6 +147,35 @@ func cliEnvPromptValidation(defaults *CLIEnvConfig) *cr.PromptValidation {
 	}
 }
 
+func validateOperatorEndpoint(endpoint string) (string, error) {
+	url, err := cr.GetURLValidator(false, false)(endpoint)
+	if err != nil {
+		return "", err
+	}
+
+	req, err := http.NewRequest("GET", url+"/isthiscortex", nil)
+	if err != nil {
+		return "", errors.Wrap(err, "verifying operator endpoint", url)
+	}
+	req.Header.Set("Content-Type", "application/json")
+	bodyBytes, err := _apiClient.MakeRequest(req)
+	if err != nil {
+		exit.Error(ErrorInvalidOperatorEndpoint(url))
+	}
+
+	var responseStr string
+
+	err = json.Unmarshal(bodyBytes, &responseStr)
+	if err != nil {
+		exit.Error(ErrorInvalidOperatorEndpoint(url))
+	}
+	if responseStr != "ok" {
```

Comment:
maybe just check status code of the response? Just a thought, it could go either way